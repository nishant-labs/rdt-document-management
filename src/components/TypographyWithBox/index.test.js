import React from 'react';
import { shallow } from 'enzyme';
import { TypographyWithBox } from './index';

const setupTest = (setupProps = {}) => {
	const baseProps = {
		type: 'h1',
		text: 'This is body of the text',
	};
	const props = { ...baseProps, ...setupProps };
	const wrapper = shallow(<TypographyWithBox {...props} />);
	return {
		wrapper,
	};
};

describe('components/TypographyWithBox', () => {
	test('should render', () => {
		const { wrapper } = setupTest();
		expect(wrapper.debug()).toMatchSnapshot();
	});
});
