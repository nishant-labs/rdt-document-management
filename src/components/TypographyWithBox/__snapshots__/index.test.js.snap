// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`components/TypographyWithBox should render 1`] = `
"<Styled(MuiBox) display=\\"flex\\" flexDirection=\\"row\\" justifyContent=\\"center\\" p={1}>
  <WithStyles(ForwardRef(Typography)) variant=\\"h1\\" gutterBottom={true}>
    This is body of the text
  </WithStyles(ForwardRef(Typography))>
</Styled(MuiBox)>"
`;
